### ISSUE TITLE
Is it intended behaviour that the right hand sidebar is dimmed?

### ISSUE THREAD

**Issue by 26tajeen** [spmbt/googleSearchExtraButtons#14]
Just asking if this is intended behaviour that the Google RHS is dimmed until you hover over it? This is the spot where Google Business results appear. I couldn't find this listed anywhere as a feature so wondered if it was a bug? This is my first time commenting/raising an issue anywhere on GitHub so go easy on me! ;-)

**Comment by spmbt** [comment]
Yes, this right sidebar contains advertisement usually, therefore, it made little contrast.
If it contain important info, we may not make transparent by special class of such message. Hint me class name (CSS) jf messages of Google Business, please (or how to repeat it by me?).

**Comment by 26tajeen** [comment]
xpdopen ?

**Comment by spmbt** [comment]
This string hide promo-blocks on the right sidebar:

.rhsvw{opacity:.16; transition:.4s}

.xpdopen class present in all .rhsv block. We need to select unical property of Google Business spot and exclude this useful block from anotheer advertisement.

In oher way, I may set option in settings, which disable all suppress of advertising...

**Comment by 26tajeen** [comment]
This is still an issue for me and it seems like a bug rather than a feature because if you search for a business, you're prevented from easily viewing the information. I solve it by removing these lines:

```
+div.gb_g[aria-label="promo"],.pdp-psy.og-pdp, .gb_Sc.gb_g .gb_ha, .gb_g.gb_ha:not(.xpdopen ){display:none}'
		+'.xpdopen{display:block!important}.rhsvw{opacity:.16; transition:.4s}.rhsvw:hover{opacity:1}

```
